developer/system/template-manager.md

```diff
@@ -0,0 +1,259 @@
+---
+layout: doc
+title: Template Manager
+permalink: /doc/template-manager/
+---
+
+Template Manager Design
+=======================
+
+Motivation
+----------
+
+Previously, template VMs were distributed by RPM packages and managed by `yum`/`dnf`.
```

Comment:
Please use terminology exactly as it appears in the [glossary](https://www.qubes-os.org/doc/glossary/), e.g., `TemplateVM` rather than `template VM`.

Reply:
Fixed.

Just to make sure, is the usage of "template" throughout the document okay?

Reply:
Yes, as explained in the glossary, that's fine.